mne/datasets/limo/limo.py

```diff
@@ -0,0 +1,253 @@
+        _fetch_file(url, destination, print_destination=False)
+        do_unzip = True
+
+    # decompress if necessary
+    if is_zip and do_unzip:
+        z1 = zipfile.ZipFile(destination)
+        files = z1.namelist()
+        if do_unzip:
+            stdout.write('Decompressing %g files from\n'
+                         '"%s" ...' % (len(files), destination))
+            z1.extractall(limo_dir)
+            stdout.write(' [done]\n')
+        z1.close()
+        # list files in .zip-folder and look for further .zip-folders to decompress
+        extracted_files = [op.join(limo_dir, f) for f in files]
+        sub_files = [e for e in extracted_files if ".zip" in e or ".tar" in e]
+
+        if sub_files:
+            stdout.write('Decompressing the extracted files ...\n')
+            for i in sub_files:
+                if '.zip' in i:
+                    sub_dir = op.splitext(i)[0]
+                    z2 = zipfile.ZipFile(i)
+                    z2.extractall(limo_dir)
+                    z2.close()
+                    # continue decompressing if necessary
+                    sub_sub_files = [j for j in os.listdir(sub_dir) if '.zip' in j]
+                    for k in sub_sub_files:
+                        z3 = zipfile.ZipFile(os.path.join(sub_dir, k))
+                        z3.extractall(sub_dir)
+                        z3.close()
+                elif '.tar' in i:
+                    t = tarfile.open(os.path.join(limo_dir, i))
+                    t.extractall(limo_dir)
+            stdout.write(' [done]\n')
+
+    # update default if desired
+    _do_path_update(path, update_path, key, name)
+    return limo_dir
+
+
+@verbose
+def load_data(subject, path=None, interpolate=False, force_update=False, update_path=None,
+              url=None, verbose=None):  # noqa: D301
+    """Fetch subjects epochs data for the LIMO data set.
+    
+    Parameters
+    ----------
+    subject : int | str
+        Subject to use. Can be in the range from 1 to 18.
+        If string, must be 'S1', 'S2' etc.
+    path : str
+        Location of where to look for the LIMO data.  
+        If None, the environment variable or config parameter
+        ``MNE_DATASETS_LIMO_PATH`` is used. If it doesn't exist, the
+        "~/mne_data" directory is used.
+    interpolate : bool
+        Whether to interpolate missing channels.
+    force_update : bool
+        Force update of the dataset even if a local copy exists.
+    update_path : bool | None
+        If True, set the MNE_DATASETS_LIMO_PATH in mne-python
+        config to the given path. If None, the user is prompted.
+    url : str
+        The location from where the dataset should be downloaded, if not
+        found on drive.
+    %(verbose)s
+    
+    Returns
+    -------
+    epochs : MNE Epochs data structure
+        The epochs.
+    """  # noqa: E501
+    if url is None:
+        url = 'http://datashare.is.ed.ac.uk/download/DS_10283_2189.zip'
```

Comment:
I personally prefer ` url = 'http://datashare.is.ed.ac.uk/download/DS_10283_2189.zip' if url is None else url`

Reply:
Might not actually be better in this case.

Reply:
I thought this would make the statement more explicit. I'll leave it as it is for now, but i'm fine with your suggestion if you think it's better.